### ISSUE TITLE
Limit elements positioning to left mouse button.

### ISSUE THREAD

**Issue by weisJ** [MyLibreLab/MyLibreLab#35]
When moving elements one should only be able to do so using the left mouse button.
For example if one uses the right button to open the context menu and keeps the button pressed as soon as the
mouse leaves the menu the elements in the background makes a big jump.

**Comment by alehel** [comment]
I'd like to tackle this one if I may.

**Comment by alehel** [comment]
I messed up trying to combine my two commits into one commit. Somehow, instead of combining the two  I somehow ended up with a total of 4 commits on my PR. I tried using _git rebase -i HEAD~2_ where I squased my autostyleApply commit into my first commit. I then pushed it to my forks branch. That somehow resultet in there being 4 commits rather than 1. Did I use the wrong git command for what I was trying to achieve?

Anyway. As the code change was so minor I decided to just do it over 😆
